uktrade/enquiry-mgmt-tool
# Set 'data hub company search page' url in env vars

Currently when editing the Data Hub Company field in an enquiry, the link to create a company in Data Hub is set to the dev environment. This PR creates an environment variable for `DATA_HUB_CREATE_COMPANY_PAGE_URL`, the value of which is passed to the `enquiry_company_search` template. 

I also set the link to open in a new tab and added hint text to the link indicating this.

I have added this variable to vault, with each environment pointing to the corresponding data hub environment (e.g. enquiry management dev would point to `https://www.datahub.dev.uktrade.io/companies/create`.

**Test instructions**

1. Set the `DATA_HUB_CREATE_COMPANY_PAGE_URL` variable (see sample env)
2. Open an enquiry
3. Click Edit
4. Click ‘search for datahub company’
5. Click ‘I can’t find what I'm looking for’
6. Click ‘I still can’t find what I’m looking for’
7. Whatever url you set as the variable should open

![Screenshot 2020-07-02 at 18 00 43](https://user-images.githubusercontent.com/22460823/86389256-f4b2f180-bc8d-11ea-9587-db3160fb3d65.png)
